specs/binding-tls.spec/src/main/scripts/io/aklivity/zilla/specs/binding/tls/config/client.alpn.default.yaml

```diff
@@ -34,5 +34,4 @@ bindings:
                 - serverca
             alpn:
                 - protocol2
-                - null
```

Comment:
I need to double check this again, I think it is trying to say what if no `alpn` protocol is negotiated, saying that it is still fair to proceed in the generic case.

Reply:
I am good with either, I will update the docs to include this option if it needs to remain.